### PR TITLE
mgmt, applicationinsights, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to [issue#35316](https://github.com/Azure/azure-sdk-for-java/issues/35316)

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/applicationinsights/azure-resourcemanager-applicationinsights/src/test/java/com/azure/resourcemanager/applicationinsights/ApplicationInsightsManagerTests.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                        .withRegion(REGION)
+                        .withExistingResourceGroup(resourceGroupName)
+                        .withSku(new WorkspaceSku().withName(WorkspaceSkuNameEnum.PER_GB2018))
+                        .withFeatures(new WorkspaceFeatures().withEnableLogAccessUsingOnlyResourcePermissions(true))
+                        .withWorkspaceCapping(new WorkspaceCapping().withDailyQuotaGb(-1D))
+                        .withRetentionInDays(30)
+                        .withPublicNetworkAccessForIngestion(com.azure.resourcemanager.loganalytics.models.PublicNetworkAccessType.ENABLED)
+                        .withPublicNetworkAccessForQuery(com.azure.resourcemanager.loganalytics.models.PublicNetworkAccessType.ENABLED)
+                        .create()
+                        .id()
```
Comment (weidongxu-microsoft) [inline sdk/applicationinsights/azure-resourcemanager-applicationinsights/src/test/java/com/azure/resourcemanager/applicationinsights/ApplicationInsightsManagerTests.java:109]: It is better not to create resource nested here.

Just have a line before `applicationInsightsManager` to do this.

**Review by weidongxu-microsoft — COMMENTED** [review]
@XiaofeiCao to review as well.

File: sdk/applicationinsights/azure-resourcemanager-applicationinsights/src/test/java/com/azure/resourcemanager/applicationinsights/ApplicationInsightsManagerTests.java (hunk 1 above)
Comment (v-hongli1) [reply sdk/applicationinsights/azure-resourcemanager-applicationinsights/src/test/java/com/azure/resourcemanager/applicationinsights/ApplicationInsightsManagerTests.java]: Fixed in the new version.

**Review by weidongxu-microsoft — APPROVED** [review]

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**